lld/test/COFF/fatlto/fatlto-invalid.s

```diff
@@ -0,0 +1,8 @@
+# RUN: llvm-mc -filetype=obj -triple=i386-pc-win32 %s -o %t
+# RUN: env LLD_IN_TEST=1 not lld-link %t /out:/dev/null /fat-lto-objects 2>&1 | FileCheck %s
+
```

Comment:
Are you able to handle a COFF error here as well? For example, when dealing with a FAT LTO .OBJ, any of the `object_error::parse_failed` returned by `COFFObjectFile::initialize`?